app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,19 @@
+require 'sinatra/json'
+
+
+# should recieve latitude, longitude, radius, price parameters
+class Server < Sinatra::Base
+  before do
+    content_type :json
+  end
+
+  get '/search' do
+    yelp = YelpService.new(params["latitude"], params["longitude"], params["radius"], params["price"])
```

Comment:
Nice job averaging the reviews from two sources.

Reply:
Sinatra has some interesting controller methods